### ISSUE TITLE
Is this code block is valid?

### ISSUE THREAD

**Issue by r-deo** [irlabamsterdam/iKAT#1]
```       
 if len(run.turns) != num_turns:
      logger.error(f'The run contains {len(run.turns)} turns, but the topic contains {num_turns} turns')
      sys.exit(255)
```
Is it valid check in main.py. Because the number of turns in a run would not be equal to the number of turns in the topic

**Comment by r-deo** [comment]
Even the official supplied JSON has an issue

 2023-08-28 20:55:36,951 ERROR __main__ The run contains 331 turns, but the topic contains 6 turns

**Comment by r-deo** [comment]
@andrewramsay

**Comment by andrewramsay** [comment]
Hi @r-deo, 

Could you provide some more information about the run file you're trying to validate? 

The intent of that code block is to ensure that a run file has the same number of turns as the selected topic in the [test topics file](https://github.com/irlabamsterdam/iKAT/blob/main/2023/data/2023_test_topics.json). So e.g. if your run is for topic number `9-1` which has 6 turns in that file, this check is supposed to be making sure that your run also has 6 turns. 

(if you'd prefer to message me about this on Google Chat instead of posting comments here, I'm at `andrew.ramsay@dcs.gla.ac.uk`)

**Comment by r-deo** [comment]
I was trying to validate 2023/scripts/baselines/runs/ret_bm25_rm3--type_manual--num_ptkb_3--k_1000--num_psg_3.official.run.json . 

The problem arises because run level turns for the test file is 331 while each of the the topic turns are different. Had msged additional details over the Gchat

**Comment by andrewramsay** [comment]
This should be fixed in #4, can you check that?

**Comment by r-deo** [comment]
It worked. Thanks
